CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.37.20 (2025-11-25)
+
+### Bug Fixes
+
+- release(snapshot) 0.37.20-SNAPSHOT ([765f6a8f](https://github.com/SAP/neonbee/commit/765f6a8f0721b8cb599d986ad0f6df813add28bc))
```

Comment:
Is this meant to still be a snapshot?

Reply:
That is a commit message. I had to change the version because it was set to 0.37.110-SNAPSHOT